app/models/user.rb

```diff
@@ -40,6 +42,8 @@ class User < ApplicationRecord
   scope :search_user, -> role, data {where("role = ? AND (phone_number = ? OR
     name LIKE ?)", role, data, "%#{data}%")}
   scope :is_online, -> {where online: true}
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Metrics/AbcSize
**Severity**: INFO
**File**: app/models/user.rb L44

Assignment Branch Condition size for send_pin is too high. [15.13/15]

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*